no-browser-site-scrapers/GreaterLawrenceFHC/index.js

```diff
@@ -0,0 +1,90 @@
+const { sites, provider } = require("./config");
+const fetch = require("node-fetch");
+const htmlParser = require("node-html-parser");
+const moment = require("moment");
+
+module.exports = async function GetAvailableAppointments(
+    _ignored,
+    fetchService = liveFetchService()
+) {
+    console.log(`${provider} starting.`);
+
+    const results = [];
+
+    for (const site of sites) {
+        const websiteData = await ScrapeWebsiteData(site, fetchService);
+
+        results.push(websiteData);
+    }
+
+    console.log(`${provider} done.`);
+    return results;
+};
+
+function liveFetchService() {
+    return {
+        async fetchAvailability(site) {
+            return await fetchAvailability(site);
+        },
```

Comment:
Huh? I do not understand what is going on here.

Reply:
The fetchService allows for injecting the mock in testing. I borrowed this "pattern" from other scraper authors (e.g., Tessa?) instead of going down the nock route of intercepting fetch requests.